packages/form-validations/src/lang.ts

```diff
@@ -0,0 +1,6 @@
+export const lang = {
+  en: {
+    required: 'This field is required',
```

Comment:
So there will be always one error message (do not count another languages) for specific form validation error?

Reply:
These will be the default ones. You can pass a lang object with your own or just with an empty string. The validations will be displayed based on their priority. Or do you want to display multiple errors at once?